src/docs/development/data-and-backend/json.md

```diff
@@ -1,159 +1,141 @@
 
 <a name="manual-encoding"></a>
-## Serializing JSON manually using dart:convert
+## dart:convert를 이용해 수동으로 JSON을 직렬화하기
 
-Basic JSON encoding in Flutter is very simple. Flutter has a built-in
-`dart:convert` library that includes a straightforward JSON encoder and
-decoder.
+Flutter 에서 기본적인 JSON 인코딩은 매우 간단합니다. Flutter 는 간단한 JSON 인코더와 디코더가
+내장된 `dart:convert` 라이브러리를 가지고 있습니다.
 
-Here is an example JSON for a simple user model.
+아래에 간단한 user 모델을 이용한 JSON 예제가 있습니다.
 
 ```json
 {
-  "name": "John Smith",
-  "email": "john@example.com"
+  "name": "홍길동",
+  "email": "gildong@example.com"
 }
 ```
 
-With `dart:convert`, you can encode this JSON model in two ways.
+`dart:convert` 를 통해 이 JSON 모델을 두가지 방법으로 인코딩 할 수 있습니다.
 
-### Serializing JSON inline
+### 인라인에서 JSON 직렬화
 
-By looking at the [dart:convert][dart:convert] documentation,
-you'll see that you can decode the JSON by calling the `jsonDecode()` function,
-with the JSON string as the method argument.
+[dart:convert][dart:convert] 의 문서를 보게 되면,
+JSON 문자열을 인자에 넣고 `jsonDecode()` 함수를 호출하여 JSON을 디코드 할 수 있다는 걸 알 수 있습니다.
 
 <!-- skip -->
 ```dart
 Map<String, dynamic> user = jsonDecode(jsonString);
 
-print('Howdy, ${user['name']}!');
-print('We sent the verification link to ${user['email']}.');
+print('안녕하세요, ${user['name']}님!');
+print('${user['email']}으로 인증 링크를 보냈습니다.');
 ```
 
-Unfortunately, `jsonDecode()` returns a `Map<String, dynamic>`, meaning
-that you do not know the types of the values until runtime. With this approach,
-you lose most of the statically typed language features: type safety,
-autocompletion and most importantly, compile-time exceptions. Your code will
-become instantly more error-prone.
+안타깝게도, `jsonDecode()` 는 `Map<String, dynamic>` 을 돌려줍니다.
+런타임 이전까지는 값의 자료형을 알 수 없게 되는 데, 그 즉슨, 대부분의
+정적 타입 언어의 기능인 타입 안전성, 자동완성, 그리고 가장 중요한 컴파일 타임 오류를 사용할 수 없게 됩니다.
+이로 인해 코드는 순식간에 오류가 발생하기 쉬운 환경에 처하게 됩니다.
+
+예를 들어 `name` 이나 `email` 필드를 접근하려 할 때, map 에 디코딩된 JSON 의 값이 있기에
+컴파일러가 탐지 할 수 없는 오류를 만들어내기 쉬워집니다.
 
-For example, whenever you access the `name` or `email` fields, you could quickly
-introduce a typo. A typo that the compiler doesn't know about since the
-JSON lives in a map structure.
+### 모델 클래스에서 JSON 직렬화
 
-### Serializing JSON inside model classes
+일반적인 모델 클래스를 사용하여 앞서 말한 문제를 방지 할 수 있습니다. 이 예제에서는 `User` 를 사용합니다.
+`User` 클래스 내에서는
 
-Combat the previously mentioned problems by introducing a plain model
-class, called `User` in this example. Inside the `User` class, you'll find:
+* `User.fromJson()` 생성자: map 구조에서 새로운 `User` 객체를 생성하기 위한 생성자
+* `toJson()` 메서드: `User` 객체를 map 구조로 변환하기 위한 메서드
 
-* A `User.fromJson()` constructor, for constructing a new `User` instance from a
-  map structure.
-* A `toJson()` method, which converts a `User` instance into a map.
+가 있습니다.
 
-With this approach, the _calling code_ can have type safety,
-autocompletion for the `name` and `email` fields, and compile-time exceptions.
-If you make typos or treat the fields as `int`s instead of `String`s,
```

Comment:
If you make typos or treat the fields as `int`s instead of `String`s,
the app won't compile, instead of crashing at runtime.

이 부분에 대한 번역이 빠진 것 같습니다